src/test/regress/expected/citus_table_triggers.out

```diff
@@ -141,12 +141,12 @@ CREATE TRIGGER test_table_delete
     FOR EACH STATEMENT
     EXECUTE FUNCTION test_table_trigger_function();
 SELECT master_get_table_ddl_events('test_table');
-                                                                                                                                                                           master_get_table_ddl_events
+                                                                                                                                                      master_get_table_ddl_events
 ---------------------------------------------------------------------
  CREATE TABLE table_triggers_schema.test_table (id integer, text_number text, text_col text)
  ALTER TABLE table_triggers_schema.test_table OWNER TO postgres
  CREATE TRIGGER test_table_delete AFTER DELETE ON table_triggers_schema.test_table FOR EACH STATEMENT EXECUTE FUNCTION table_triggers_schema.test_table_trigger_function()
- CREATE CONSTRAINT TRIGGER test_table_insert AFTER INSERT ON table_triggers_schema.test_table DEFERRABLE INITIALLY IMMEDIATE FOR EACH ROW WHEN (((new.id OPERATOR(pg_catalog.>) 5) OR ((new.text_col IS NOT NULL) AND ((new.id)::numeric OPERATOR(pg_catalog.<) to_number(new.text_number, '9999'::text))))) EXECUTE FUNCTION table_triggers_schema.test_table_trigger_function()
+ CREATE CONSTRAINT TRIGGER test_table_insert AFTER INSERT ON table_triggers_schema.test_table DEFERRABLE INITIALLY IMMEDIATE FOR EACH ROW WHEN (((new.id > 5) OR ((new.text_col IS NOT NULL) AND ((new.id)::numeric < to_number(new.text_number, '9999'::text))))) EXECUTE FUNCTION table_triggers_schema.test_table_trigger_function()
```

Comment:
Since `PushOverrideEmptySearchPath` adds `pg_catalog` into the temp `search_path`, this is the expected behavior but I'm not sure why we weren't respecting that fact before the changes done in this pr.

Reply:
This is explained here: https://github.com/citusdata/citus/pull/2147#issue-320109636 Mostly for performance reasons.

And, that's relevant for query deparsing, so shouldn't matter for DDLs